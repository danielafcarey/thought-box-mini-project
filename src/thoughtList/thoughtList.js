import React from 'react';

export const ThoughtList = ({ thoughtList }) => {

  return (
    <div>

    </div>
  );
}
