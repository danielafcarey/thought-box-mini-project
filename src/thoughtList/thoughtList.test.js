import React from 'react';
import { shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';

import { ThoughtList } from './thoughtList.js';

describe('ThoughtList', () => {

  it('should match the snapshot with no thoughts', () => {
    const mockThoughts = [];
    const thoughtList = renderer.create(<ThoughtList thoughtList={ mockThoughts }/>);

    expect(thoughtList).toMatchSnapshot();
  });

  it('should match the snapshot with thoughts', () => {
    const mockThoughts = [{ id: 'thought0', title: 'title', body: 'body' }];
    const thoughtList = renderer.create(<ThoughtList thoughtList={ mockThoughts }/>);

    expect(thoughtList).toMatchSnapshot();
  });

});
