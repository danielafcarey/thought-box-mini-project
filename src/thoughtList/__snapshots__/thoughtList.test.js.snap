// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThoughtList should match the snapshot with no thoughts 1`] = `<div />`;

exports[`ThoughtList should match the snapshot with thoughts 1`] = `<div />`;
